extensions/adobe/experience/offer-management/offer-placement.schema.json

```diff
@@ -0,0 +1,61 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id":
+    "https://ns.adobe.com/experience/offer-management/offer-placement",
+  "type": "object",
+  "title": "Placement",
+  "description": "Offer Placement",
+  "definitions": {
+    "offer-placement": {
+      "properties": {
+        "@id": {
+          "type": "string",
+          "format": "uri",
+          "title": "ID",
+          "description": "The unique identifier of the placement"
+        },
+        "xdm:name": {
+          "type": "string",
+          "title": "Name",
+          "description": "Offer placement name"
+        },
+        "xdm:description": {
+          "type": "string",
+          "title": "Description",
+          "description": "Offer placement description"
+        },
+        "xdm:channel": {
```

Comment:
@kstreeter does this map to the channel stuff we are still working on?

Reply:
Now it is a string holding a `@id` of a channel XDM ( `http&#58;s://ns.adobe.com/xdm/channels/channel`) instance